command_test.go

```diff
@@ -89,3 +108,49 @@ Commands:
 		assert.Equal(t, t.Name(), expected, got)
 	})
 }
+
+func TestSubCmdHelpOutput(t *testing.T) {
+	withNested := `Usage: mockSubCmd Test a subcommand.
+
+Aliases: s, sc, sub
+
+Description: A simple mock subcommand with aliases and its own subcommand.
+
+Commands:
+	s, sc, sub, mockSubCmd  - A simple mock subcommand with aliases and its own subcommand.
+`
+
+	noNested := `Usage: mockSubCmdNoNested Used for help output tests.
+
+Aliases: s, sc, sub
+
+Description: A simple mock subcommand with aliases and no nested subcommands.
+`
+
+	for _, test := range []struct {
+		cmd            Command
```

Comment:
not very important, but it might be nice to use the same order of fields in the type and values.